lib/common/output_text.c

```diff
@@ -14,6 +14,7 @@
 
 #include <crm/crm.h>
 #include <crm/common/output_internal.h>
+#include <crm/common/strings_internal.h>
```

Comment:
This should just include crm_internal.h (before anything else) rather than these three. That also ensures it respects deprecation.